### PR TITLE
Fix TFLPOS and model size computation

### PR DESCRIPTION
**Pull request by ver217**
1. Fix colossalai utils with sharded model v2
2. Fix TFLPOS and model size computation

### CODE REVIEW

File: zero/common/gpt2.py [hunk 1]
```diff
@@ -197,5 +197,6 @@ def gpt2_builder():
 
     CONFIG['dataset'] = os.environ['DATA']
     CONFIG['tokenizer'] = os.environ['TOKENIZER']
+    CONFIG['model']['numel'] = get_model_size(build_model())
```
Comment (FrankLeeeee) [inline zero/common/gpt2.py:200]: This will cause model to be built twice?

Comment (FrankLeeeee) [reply zero/common/gpt2.py:200]: Is it because of zero init context?

Comment (ver217) [reply zero/common/gpt2.py:200]: Yes, we can also compute model size locally and then do all-reduce. But the size may have bias due to padding.

Comment (FrankLeeeee) [reply zero/common/gpt2.py:200]: ok

Comment (feifeibear) [reply zero/common/gpt2.py:200]: I can get model size in zero init context.

**Pull request merged**